src/main/resources/templates/team/main.html

```diff
@@ -53,7 +53,7 @@ <h2>Tým:</h2>
         </tr>
         <tr>
             <td></td>
-            <td><input type="submit" th:value="${action=='/team/add'?'Nový':'Uložit'}"/>
+            <td><input type="submit" th:value="${action=='/team/main'?'Nový':'Uložit'}"/>
```

Comment:
Asi neni nutna zmena, ale asi bych pridal mezery za ternarni operator. `${action=='/team/main'? 'Nový': 'Uložit'}`. Necham na tvoje posouzeni, jestli to v tom HTML vypada prehlednej nebo ne. V Jave bech tot ak chtel v HTML je mi to jedno ;)

Reply:
vypada to stejne, ale mezeru jsem pridal